src/openrct2/ride/coaster/AirPoweredVerticalCoaster.cpp

```diff
@@ -929,6 +929,27 @@ static void air_powered_vertical_rc_track_booster(
     paint_util_set_general_support_height(session, height + 32, 0x20);
 }
 
+static void air_powered_vertical_rc_track_on_ride_photo(
```

Comment:
What did you use as a base to implement this function?

Reply:
I used the APVC's straight track pieces and the splash boats' on-ride photo as a base.